### PR TITLE
Split up trends charts into different charts

### PR DESCRIPTION
**Pull request by liquidx**
Attempt to address #451 and #447 

Refactor DailyIncrease component into something that can be reused for the Active, Deceased and Tested charts. 
Replace total trends chart with the DailyIncrease charts showing Active, Deceased and Tested.
Default to showing 6 months in the large charts, add option to show 3 months or 6 months (all time).
Fix rendering artifacts when showing 6 months of data.
Comment out the initial chart rendering code as it doesn't seem to be called.

### CODE REVIEW

**Comment by liquidx** [comment]
Screenshot:

Screen Shot 2020-11-21 at 23 46 54
Screen Shot 2020-11-21 at 23 47 38

File: src/index.js [hunk 1]
```diff
@@ -257,6 +308,7 @@ const loadDataOnPage = () => {
       ddb.totals = newTotals[0];
       ddb.totalsDiff = newTotals[1];
       ddb.trend = summaryData.daily;
+      console.log(ddb.trend);
```
Comment (reustle) [inline src/index.js:311]: Maybe we can get rid of this when we deploy?

File: src/index.js [hunk 2]
```diff
@@ -168,18 +174,33 @@ const setLang = (lng) => {
...
-          CHART_TIME_PERIOD
-        );
-        dailyIncreaseChart = drawDailyIncreaseChart(
-          ddb.trend,
-          dailyIncreaseChart,
-          LANG,
-          CHART_TIME_PERIOD
-        );
+
+        // TODO: This doesn't seem to be needed and can be removed.
```
Comment (reustle) [inline src/index.js:178]: Maybe it's safe to remove this too

**Comment by reustle** [comment]
This looks great! Thanks for putting this together, it's definitely something that people have been looking forward to seeing.

File: src/index.js (hunk 1 above)
Comment (liquidx) [reply src/index.js:311]: Good catch, fixing.

File: src/index.js (hunk 2 above)
Comment (liquidx) [reply src/index.js:178]: I'm thinking of removing this in a separate change, I'm trying to figure out if there's a corner case we used to have which we kept this around for.

**Pull request merged**